CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+## Git Workflow
+This project will follow the [Gitflow Workflow].
+
+## Pull Request Process
+1. When your feature/fix is complete create a pull request to merge your barnch
+into `develop`
+2. Use a title that reflects the work done and a summary to give context on the change [^2].
+  If it is a visual change, including a screenshot may be helpful.
+3. Request the review of relevant developers. A minimum of two approving reviews
+  are required.
+
+## Styleguides
+### Git Commit Messages [^1]
+1. Separate subject from body with a blank line
+2. Limit the subject line to 72 characters or less
+3. Capitalize the subject line
+4. Do not end the subject line with a period
+5. Use the present tense ("Add feature" not "Added feature") in the subject line
+6. Wrap the body at 72 characters
+7. Use the body to explain what and why vs. how
+
+### Branch Names
+Use the convention `my-new-feature-#62`
```

Comment:
Can we maybe add a convention for if a branch isn't for a specific issue just to clarify all cases?
eg. my-new-feature

Reply:
Addressed branch names in new commit :+1: